core/src/main/scala/kafka/coordinator/PidMetadata.scala

```diff
@@ -19,10 +19,17 @@ package kafka.coordinator
 import kafka.utils.nonthreadsafe
 
 @nonthreadsafe
-private[coordinator] class PidMetadata(val pid: Long) {
+private[coordinator] class PidMetadata(val pid: Long,
```

Comment:
@hachikuji How about `k.coordinator.transaction`? Then I will just create a a subfolder under `coordinator` and we can consider also moving consumer stuff to `k.c.group`.